src/robusta/integrations/receiver.py

```diff
@@ -189,7 +200,7 @@ def on_open(self, ws):
         )
         ws.send(json.dumps(open_payload))
 
-    def __validate_request(self, action_request: ExternalActionRequest, validate_timestamp: bool) -> bool:
+    def __validate_request(self, action_request: ExternalActionRequest, validate_timestamp: bool) -> ValidationResponse:
```

Comment:
if makes more sense to me that validate request will throw an exception in case it fails (and return None), as most of the time the request should be valid. "ValidationResponse" looks like an exception class.

Reply:
I want to return different permutations of http_code and error_code
I think a return value is better here